scripts/local_test.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
wat

Seriously `/bin/bash` is a thing you can rely on. or `/bin/sh` even - this is all posix shell...